examples/fmri_spm_auditory.py

```diff
@@ -372,14 +384,14 @@ def getstripdir(subject_id):
      [('analysis.contrastestimate.con_images', 'contrasts.@con'),
       ('analysis.contrastestimate.spmT_images', 'contrasts.@T')]),
 ])
+
 """
 Execute the pipeline
 --------------------
-
 The code discussed above sets up all the necessary data structures
 with appropriate parameters and the connectivity between the
 processes, but does not generate any output. To actually run the
-analysis on the data the ``nipype.pipeline.engine.Pipeline.Run``
+analysis on the data the :func:`nipype.pipeline.engine.Workflow.run`
```

Comment:
This doesn't link.

Reply:
Yeah, I was bugged by this, but gave up at some point. I'm sure there is a way to make it link, but I didn't find it in a first effort.

Reply:
Okay, it links now. We should have a second pass through all the examples because they attempt to link many inexistent objects (plus interfaces, which should be referenced as regular rst links).